docs/using/installing/index.rst

```diff
@@ -4,17 +4,27 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The following diagram illustrates the architecture of the cluster you are about to install:
 
-There is also a ``clusterhq-flocker-node`` package which is installed on each node in the cluster.
-It contains the services that need to run on each node.
+.. image:: images/installintro.png
+   :width: 75%
+   :alt: Diagram illustrating the Flocker CLI and control service and agents.
 
-.. note:: The ``clusterhq-flocker-node`` package is pre-installed by the :ref:`Vagrant configuration in the tutorial <tutvagrant>`.
+The CLI is installed on your local machine via the ``flocker-cli`` package which provides command line tools to control the cluster. 
+This package also includes the ``flocker-ca`` tool, which you must use to generate certificates for all the Flocker components.
+
+The Flocker agents are installed on any number of hosts (or nodes) in the cluster where your containers will run.
+The agent software is included in the ``clusterhq-flocker-node`` package.
+
+There is also a Flocker control service which you must install on one of the agent hosts, or on a separate machine. 
+The control service is also included in the ``clusterhq-flocker-node`` package, but is activated separately later in these installation instructions.
+
+.. note:: The agents and control service are pre-installed by the :ref:`Vagrant configuration in the tutorial <tutvagrant>`.
```

Comment:
@davidoliverSP2 Great to have a style guide. We used to have one for our old product's documentation. It was by no means comprehensive, but I'll send it to you in a private message, I think it is a good place to start.

In Flocker's Python code, we have style preferences but many of them are enforced as you know by the flake8 lint builder. This means that a build won't pass unless certain style rules are adhered to. I have proposed in the past (on JIRA I thought but I can't find it) that we use a tool similar to flake8 but for restructured text. I haven't yet tried any myself but https://pypi.python.org/pypi/restructuredtext_lint and https://pypi.python.org/pypi/doc8 exist. Having automatic linting means faster and less tedious review cycles.